### DISCUSSION TITLE
Dsi wifi "works"

### DISCUSSION THREAD

**Discussion by ghost** [rafaelvcaetano/melonDS-android#753] [Ideas]
Without proxy in dsi advanced settings the user agreement errors out with 020100
With one it errors out instead with 312010
That might help you figure out wifi connection

**Comment by Damaj301damaj-lol** [comment]
Error 2 is invalid.

**Reply by ghost** [reply]
What do you mean?

**Reply by Damaj301damaj-lol** [reply]
> What do you mean?

It has no meaning, if you try to use the code checker, it would come out as invalid
